src/cred.c

```diff
@@ -549,6 +549,15 @@ fido_cred_clean_attstmt(fido_attstmt_t *attstmt)
 	memset(attstmt, 0, sizeof(*attstmt));
 }
 
+static void
+fido_cred_clean_attobj(fido_cred_t *cred)
```

Comment:
Would it make sense for `fido_cred_reset_rx()` to call this function?

Reply:
Yes, that makes sense. Pushed it.